C4SGSvc/build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.cs4g'
+version = '0.0.1-SNAPSHOT'
+
+description = """C4SG"""
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+tasks.withType(JavaCompile) {
+	options.encoding = 'UTF-8'
+}
+
+configurations.all {
+    exclude group: 'org.springframework.boot', module: 'spring-boot-starter-tomcat'
+    exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
+    exclude group: 'org.springframework.boot', module: 'logback-classic'
+}
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+dependencies {
+    compile(group: 'org.springframework.boot', name: 'spring-boot-starter-web', version:'1.4.2.RELEASE') {
+exclude(module: 'logback-classic')
+    }
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc', version:'1.4.2.RELEASE'
```

Comment:
Should we move spring-boot, log4j, swagger general version numbers into property block?
E.g. in gradle-wrapper.properties or in build.gradle
`ext {
  springbootVersion:'1.4.2.RELEASE',
  ...
}
`
Then use like 
`compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: ext.springbootVersion
`
etc